lib/views/quiz/quiz_unlock.dart

```diff
@@ -106,8 +106,8 @@ class _QuizUnlockState extends State<QuizUnlock> {
         ? GridView.count(
             crossAxisCount: 2,
             children: [
-              getTopHalf(),
               getBottomHalf(),
+              getTopHalf(),
```

Comment:
Do you wanna rename so that the top half is called top half?

Reply:
I don't have better names for this. Sam g wanted the order to be changed on landscape which is why it is like this now.  He reckons that the main focus is the lhs of the screen and shouldd have the unlock code widget

Reply:
Okay, that's fine